core/src/broadcast_stage.rs

```diff
@@ -79,6 +82,8 @@ impl BroadcastStageType {
         retransmit_slots_receiver: RetransmitSlotsReceiver,
         exit_sender: &Arc<AtomicBool>,
         blockstore: &Arc<Blockstore>,
+        leader_schedule_cache: &Arc<LeaderScheduleCache>,
+        bank_forks: &Arc<RwLock<BankForks>>,
```

Comment:
`&Arc<...>` is redundant. It should be either `&` or `Arc`.
see https://github.com/solana-labs/solana/pull/14234 for example.
Our codebase is rife with these though, so this is just fyi for future changes.

Reply:
leaving as-is for this change